1756697407322	Sync.LogManager	DEBUG	Flushing file log
1756697407323	Sync.LogManager	DEBUG	Log cleanup threshold time: 1755833407323
1756697407324	Sync.LogManager	DEBUG	Done deleting files.
1756698025616	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1756698025616	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1756698025616	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1756698025617	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1756698029648	addons.xpi	ERROR	System addon update list error Error: Failed downloading XML, status: 0, channelStatus: 2152398878, errorCode: 2, reason: error
1756698054051	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1756698054051	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1756698054054	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1756698054054	Sync.Service	DEBUG	User-Agent: Firefox/142.0 (Linux x86_64) FxSync/1.144.0.20250818081258.desktop
1756698054054	Sync.Service	INFO	Starting sync at 2025-09-01 05:40:54 in browser session YwKi1OBOWzel
1756698054054	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1756698054056	Sync.Status	INFO	Resetting Status.
1756698054056	Sync.Status	DEBUG	Status.service: error.sync.failed_partial => success.status_ok
1756698054056	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1756698054056	FirefoxAccounts	DEBUG	getOAuthToken enter
1756698054056	FirefoxAccounts	TRACE	getCachedToken returning cached token
1756698054056	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1756698054057	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1756698075199	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/197106278/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1756698075199	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1756698075199	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1756698075199	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1756698075200	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1756698075200	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 6584850 ms.
1756698075200	Sync.SyncScheduler	DEBUG	Next sync in 6584850 ms. (why=client-backoff-schedule)
1756698075201	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
